src/renderer/settings/plugin-settings.ts

```diff
@@ -16,4 +16,5 @@ export enum PluginSettings {
     ColorConverter = "Color Converter",
     BrowserBookmarks = "Browser Bookmarks",
     Dictionary = "Dictionary",
+    ControlPanel = "Control Panel",
```

Comment:
Please remove your Plugin from `src\renderer\settings\plugin-settings.ts` and add it to `src\renderer\settings\settings-os-specific.ts` with a `win:` prefix. This makes your plugin settings only visible on Windows.

Reply:
Do you mean `win32:` as prefix? The other Windows-only plugins use this, too.

Reply:
Yes I mean `win32:`, sorry my bad